core/src/fi/jarno/sumblocks/resurrection/Actors/GameBoard.java

```diff
@@ -35,9 +36,7 @@
 
     private Vector2 _touch = new Vector2(),
                     _drag = new Vector2(),
-                    _delta = new Vector2(),
-                    _srcGridPos = new Vector2(),
```

Comment:
src and dst gridPos would get their values changed randomly, if user was tapping or doing some multitouching while swap was ongoing that would cause a result of wrongly changed grid positions to blocks.